assets/blocks/course-outline/outline-block/existing-lessons-modal/lessons-modal-style.scss

```diff
@@ -0,0 +1,102 @@
+.wp-block-sensei-lms-course-outline__existing-lessons-modal {
+	min-width: 566px;
+	max-width: 697px;
```

Comment:
I think it would be nice to also set some height in the modal or in somewhere else to avoid the jumps while interacting, WDYT?

https://github.com/Automattic/sensei/assets/876340/ed289b0b-ec40-4ef6-9ed9-3c5d5dd17566

Reply:
Fixed here: https://github.com/Automattic/sensei/commit/9e1d6ed78fafd4d2e4e94b51bfd935e3d66fa3d5